libs/shared/src/types/events/index.ts

```diff
@@ -33,7 +33,7 @@ export interface IEmailOptions {
   payloadDetails?: any;
   notificationDetails?: any;
   ipPoolName?: string;
-  customData?: Record<string, Record<string, unknown>>;
+  customData?: Record<string, any>;
```

Comment:
why is this change needed?

Reply:
This change is from old PR. https://github.com/novuhq/novu/pull/4504
But this PR was towards main so I included that commit in this PR to make it work

Reply:
I see, let's merge it for now but we could find another way to do so without `any` there.